docs/integrations/security-threat-detection/zscaler-private-access.md

```diff
@@ -9,121 +9,104 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+ZPA uses the LSS to stream logs from the Zscaler service and delivers them to the Sumo Logic Hosted Collector via Syslog.
 
-1. Configure Sumo Logic Hosted Collector and a Cloud Syslog Source.
-2. Configure App Connector in ZPA.
-3. Deploy an App Connector on a Supported Platform.
-4. Configure Log Receivers in ZPA to send logs to Sumo Logic.
+LSS is deployed using two components, a log receiver and a ZPA App Connector. LSS resides in ZPA and initiates a log stream through a ZPA Public Service Edge (formerly Zscaler Enforcement Node or ZEN). The App Connector resides in your company's enterprise environment. It receives the log stream and then forwards it to Sumo Logic Cloud Syslog.
 
-### Configure Sumo Logic hosted collector and a Cloud Syslog source
+To collect logs for ZPA, follow the below steps in Sumo Logic.
 
-To collect logs for ZPA, do the following in Sumo Logic:
+### Step 1: Configure Sumo Logic hosted collector and a Cloud Syslog source
 
-1. Configure a [Hosted Collector](/docs/send-data/hosted-collectors).
-2. Perform the steps in[ Configure a Cloud Syslog Source](/docs/send-data/hosted-collectors/cloud-syslog-source#configure-a-cloudsyslogsource). and configure the following Source fields:
+1. Configure a [Sumo Logic Hosted Collector](/docs/send-data/hosted-collectors).
+2. Perform the steps in [Configure a Cloud Syslog Source](/docs/send-data/hosted-collectors/cloud-syslog-source#configure-a-cloudsyslogsource) and configure the following Source fields:
     * **Name**. (Required) A name is required. Description is optional.
-    * **Source Category**. (Required) [Provide a realistic Source Category example for this data type]. The Source Category metadata field is a fundamental building block to organize and label Sources. For details, see [Best Practices](/docs/send-data/best-practices).
+    * **Source Category**. (Required) Provide a realistic Source Category example for this data type. The Source Category metadata field is a fundamental building block to organize and label Sources. For details, see [Best Practices](/docs/send-data/best-practices).
 3. In the Advanced section, specify the following configurations:
-    * **Enable Timestamp Parsing**. True
-    * **Time Zone**. Use time zone from log file. If none is detected use: Use Collector Default.
-    * **Timestamp Format**. Auto Detect
-4. In the Processing Rules for Logs section, add a Processing Rule:
-    * **Name:** `Remove Syslog String`
-    * **Filter**: `(\<\d+\>1 - - - - - - \>\{)`
-    * **Type**: `Mask messages that match`
-    * **Mask String**: `{`
+    * **Enable Timestamp Parsing**. True.
+    * **Time Zone**. Use time zone from log file. If none is detected, use: Use Collector Default.
+    * **Timestamp Format**. Auto Detect.
+4. In the **Processing Rules for Logs** section, add a Processing Rule:
+    * **Name:** `Remove Syslog String`.
+    * **Filter**. `(\<\d+\>1 - - - - - - \{)`.
```

Comment:
@gab-gabriel, thanks again for providing the fix [here](https://github.com/SumoLogic/sumologic-documentation/pull/4207). Would this regex `(\<\d+\>1 - - - - - - \{)` match `<165>1 - - - - - - <Syslog Token>`?

Reply:
@gab-gabriel , I'm going to go ahead and merge this. If you have any edits, please reply to this thread.

Reply:
@kimsauce I don't think this will work, since what shows up in the logs is `<165>1 - - - - - - <>`. What you have is what the Zscaler docs say to do, though, but I think it's wrong there too